packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -227,6 +229,23 @@ class Dialog extends AutoControlledComponent<WithAsProp<DialogProps>, DialogStat
     }
   };
 
+  /**
+   * It's needed to control after state changes because self controlled dialogs
+   */
+  componentDidUpdate() {
```

Comment:
Please could you add tests for this logic?

Reply:
e2e tests added